### PR TITLE
Small improvements to comments

### PR DESCRIPTION
**Pull request by double-beep**
I have made some small improvements to the comments here, removing unneeded comments and details and expanding some also.

### CODE REVIEW

**Comment by gunr2171** [comment]
Mind squashing your commits into one? You've got a lot of commits [with the same name](https://xkcd.com/1296/).

File: sites/stackoverflow.com/answers.md [hunk 1]
```diff
@@ -44,13 +44,13 @@ While this link might provide some limited, immediate help, an answer [should in
 Please don't post answers on obviously off topic / bad questions! [See: **Should one advise on off topic questions?**](//meta.stackoverflow.com/q/276572)
 
 ###[A] Tool recommendation for a tool request
-Thank you for participating to Stack Overflow, however please note that the question you have answered is off-topic as per the [help/on-topic] since it is a tool request. Answers to such questions tend to be of [poor quality](//meta.stackexchange.com/a/8259) and quickly become stale (what might have been a good resource two years ago may simply not exist anymore). You can flag this question as off-topic by clicking the "flag" link once you have sufficient reputation.
+Thank you for participating to Stack Overflow. However please note that the question you have answered is off-topic as per the [help/on-topic] since it is a tool request. Answers to such questions tend to be of [poor quality](//meta.stackexchange.com/a/8259) and quickly become stale (what might have been a good resource two years ago may simply not exist anymore). You can flag this question as off-topic by clicking the "flag" link once you have sufficient reputation.
```
[the remark this answers is not in the document]
Comment (double-beep) [reply sites/stackoverflow.com/answers.md]: ok, I will commit that, but anyway I have ticked allow edits from maintainers, so you can, too.

File: sites/stackoverflow.com/answers.md [hunk 2]
```diff
@@ -44,13 +44,13 @@ While this link might provide some limited, immediate help, an answer [should in
...
 
 ###[A] Pointing out a typo
 Please don't post answers only pointing out a typographical issue or a missing character. Such answers are unlikely to help future visitors since they are specific to OP's code. Instead, flag or vote to close the question as off-topic as per the [help/on-topic].
 
 ###[A] Code as image
-[Please don't post your code as an image.](//meta.stackoverflow.com/q/285551)
+[Please don't post your code as an image.](//meta.stackoverflow.com/q/285551).
```
[the remark this answers is not in the document]
Comment (double-beep) [reply sites/stackoverflow.com/answers.md]: You are right, I will commit that!

**Comment by double-beep** [comment]
Feel free to Squash & Merge the commits. I was lazy squashing all of them in one, or in three as there are three sites. BTW, I am thinking of adding some comments, but I am going to do another PR for it.

File: sites/stackoverflow.com/edit-summary-question.md [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
 ###[EQ] New question
-If you have a new question, please don't edit it over your old one. [Ask one here](//$SITEURL$/questions/ask);
+If you have a new question, please don't edit it over your old one. Ask one here: https://$SITEURL$/questions/ask;
 
 ###[EQ] Destroying content
 Please don't destroy content on $SITENAME$;
 
 ###[EQ] Tags
-The tags you have been using are not appropriate for this question. Please take the [tour], review [what are tags and how should I use them?](//$SITEURL$/help/tagging) and [edit] your post. Remember to at least read the mouseover on the tags you are using when asking a question. Thank you!
+The tags you have been using are not appropriate for this question. Please take the [tour], review what are tags and how should I use them?: https://$SITEURL$/help/tagging and [edit] your post. Remember to at least read the mouseover on the tags you are using when asking a question.
```
[the remark this answers is not in the document]
Comment (double-beep) [reply sites/stackoverflow.com/edit-summary-question.md]: @gunr2171 well, this needs cleanup: we are talking about edit summaries where markdown is not supported. I am going to remove some links or leave this as an auto comment to a question.

[the remark this answers is not in the document]
Comment (gunr2171) [reply sites/stackoverflow.com/edit-summary-question.md]: Ah, I didn't realize markdown (links) are not supported in edit summaries. This is fine as-is.

**Comment by double-beep** [comment]
@gunr2171 thanks! I am moving on and improve the comments, not grammatically, now.

**Pull request merged**